client.go

```diff
@@ -1408,6 +1408,25 @@ func linkProperty(key string, value interface{}) LinkOption {
 	}
 }
 
+// LinkSourceName sets the name of the source queue
+func LinkSourceName(name string) LinkOption {
+	return func(l *link) error {
+		l.name = name
+		return nil
+	}
+}
+
+// LinkSourceCapabilities sets the source capabilities
```

Comment:
Please add a period to the end of this sentence.

Reply:
Fixed